engine/graphics/src/vulkan/graphics_vulkan.cpp

```diff
@@ -1014,8 +1097,17 @@ namespace dmGraphics
         }
     #endif
 
+        uint16_t extensionNameCount = 0;
+        const char** extensionNames = 0;
+
+    #ifdef DM_VULKAN_VALIDATION
```

Comment:
Is this particularly expensive?
Could it be useful to have it behind a runtime flag? E.g. for quicker debugging for users